service/service.go

```diff
@@ -276,6 +283,36 @@ func setLogFormatAndLevel(logFormat log.Formatter, level log.Level) {
 	log.SetLevel(level)
 }
 
+func getStorageArrays(secretParams *viper.Viper, opts *Opts) () {
+		// Access the storagearrays key (which is a slice of maps)
```

Comment:
It is better to add a Godoc comment before the function definition. This makes it easier for someone to hover over the function when called to have an idea of what the function does. This comment itself does not add much value as it is obvious what the code will do in the next line.